facades/PC/build.gradle.kts

```diff
@@ -105,37 +98,9 @@ tasks.register<RunTerasology>("permissiveNatives") {
     systemProperty("java.library.path", rootProject.file(dirNatives + "/" + nativeSubdirectoryName()))
 }
 
-/******************************************
- * Headless server
- */
-
-apply(from="server.build.gradle")
-
-// TODO: Seems to always be up to date so no modules get copied
-tasks.register<Sync>("setupServerModules") {
-    description =
-        """Parses "extraModules" - a comma-separated list of modules and puts them into $localServerDataPath"""
-
-    val extraModules: String? by project
-    extraModules?.let {
-        // Grab modules from Artifactory - cheats by declaring them as dependencies
-        it.splitToSequence(",").forEach {
-            logger.info("Extra module: {}", it)
-            dependencies {
-                "serverModules"(group = "org.terasology.modules", name = it, version = "+")
-            }
-        }
-    }
-
-    from(configurations.named("serverModules"))
-    into(File(rootProject.file(localServerDataPath), "modules"))
-}
-
 // TODO: Make a task to reset server / game data
 tasks.register<RunTerasology>("server") {
     description = "Starts a headless multiplayer server with data stored in [project-root]/$localServerDataPath"
-    dependsOn("setupServerConfig")
-    dependsOn("setupServerModules")
     args("-headless", "-homedir=$localServerDataPath")
```

Comment:
You could add `-overrideDefaultConfig=override.cfg` here to match the headless run config for IntelliJ 👍